src/components/organize/tasks/TaskDetailsSection.tsx

```diff
@@ -82,6 +82,34 @@ const TaskDetailsCard: React.FunctionComponent<TaskDetailsCardProps> = ({
         })}
         value={task.expires && <ZetkinDateTime datetime={task.expires} />}
       />
+      <TaskProperty
```

Comment:
Is it worth refactoring `TaskProperty` as a wrapper, so you can use `<FormattedMessage />` as a child component, instead of `intl.formatMessage` as a prop

Reply:
I think that's a good idea, but I also think this is probably changing in ways we can't really imagine right now. So I would postpone any refactoring until after we know what the patterns for displaying and editing data will actually be long-term.